docs/dev-guide/installingNative.md

```diff
@@ -93,11 +95,22 @@ Most of these prerequisites can be installed using the package manager of your O
     The main prerequisites can be installed with [Homebrew](http://brew.sh/):
 
     ```sh
-    brew install git graphviz postgresql redis uv d2 node npm pgvector
+    brew install git graphviz postgresql@17 redis uv d2 node pgvector
+
     # Optional; needed only for some example questions that use LaTeX
     brew install texlive
     ```
 
+    You may want to start up the `postgresql` server on boot, and add binaries to your path:
+
+    ```sh
+    brew services start postgresql@17
+    # zsh
+    echo 'export PATH="$(brew --prefix postgresql@17)/bin:$PATH"' >> ~/.zshrc && source ~/.zshrc
+    # bash
+    echo 'export PATH="$(brew --prefix postgresql@17)/bin:$PATH"' >> ~/.bashrc && source ~/.bashrc
```

Comment:
I just reinstalled Postgres for fun and realized that I had previously fixed this with `brew link postgresql@17`. IMO that's probably better?

Reply:
Changed in 94d74ac4336856b928672dc27b716b20d4f1ba4a.